Rakefile

```diff
@@ -1 +1,6 @@
 require "bundler/gem_tasks"
+
+desc "Console"
+task :console do
+  system "bundle exec pry -r ./lib/test_changes"
+end
```

Comment:
Heh. Can you rebase this pull request without the console rake task. Please update the README too? Thanks!